examples/execute_into/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ShaileshSurya/hammer"
+)
+
+func main() {
+	// init client
+	client := hammer.New()
+
+	// build request
+	request, err := hammer.RequestBuilder().
```

Comment:
with `hammer.RequestBuilder` it is given that here we are trying to build request. We do not need this comment. 

I prefer to use comments very sparingly.

Reply:
We should remove this from execute.go

Reply:
Fixed